jvm/ml4ir-inference/pom.xml

```diff
@@ -217,7 +217,13 @@
   <dependencies>
     <dependency>
       <groupId>org.tensorflow</groupId>
-      <artifactId>tensorflow-core-platform</artifactId>
+      <artifactId>tensorflow-core-api</artifactId>
+      <version>1.0.0-rc.1</version>
```

Comment:
@mohazahran did you check if we could upgrade to 1.0.0 without upgrading on the python side? Also "rc" is usually a beta release I believe.